tests/apis/test_google_operational_info.py

```diff
@@ -0,0 +1,360 @@
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+#   Please refer to the AUTHORS file for more information.
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#   GNU Affero General Public License for more details.
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program. If not, see <https://www.gnu.org/licenses/>.
+import json
+from datetime import date, datetime
+from http import HTTPStatus
+from typing import Any, Dict
+from unittest.mock import MagicMock, patch
+
+from freezegun import freeze_time
+from pytest import fixture, mark
+from pytest_sanic.utils import TestClient
+
+from immuni_analytics.celery.authorization.tasks.verify_safety_net_attestation import (
+    _verify_safety_net_attestation,
+)
+from immuni_analytics.core import config
+from immuni_analytics.core.managers import managers
+from immuni_analytics.helpers.safety_net import get_redis_key
+from immuni_analytics.models.operational_info import OperationalInfo
+from immuni_common.models.enums import Platform
+from tests.fixtures.safety_net import POST_BODY_WITH_EXPOSURE, POST_TIMESTAMP, TEST_APK_DIGEST
+from tests.helpers.test_safety_net import _operational_info_from_post_body
+
+
+@fixture
+def headers() -> Dict[str, str]:
+    return {"Immuni-Dummy-Data": "0", "Content-Type": "application/json; charset=utf-8"}
+
+
+@freeze_time(datetime.utcfromtimestamp(POST_TIMESTAMP))
+@patch("immuni_analytics.helpers.redis._LOGGER.info")
+async def test_google_operational_info_with_exposure(
+    redis_logger_info: MagicMock,
+    client: TestClient,
+    safety_net_post_body_with_exposure: Dict[str, Any],
+    headers: Dict[str, str],
+) -> None:
+    with patch("immuni_analytics.helpers.safety_net.config.SAFETY_NET_APK_DIGEST", TEST_APK_DIGEST):
```

Comment:
Do you think you can extract this part? we have three tests with the same body 🤔

Reply:
I would keep it as is to show the actual flow (even if mocked)